### PR TITLE
[16][MIG] partner vat unique

### PR DESCRIPTION
**Pull request by AnizR**
Migration of partner_vat_unique

### CODE REVIEW

File: partner_vat_unique/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from odoo.exceptions import ValidationError
+from odoo.tools import config
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    vat = fields.Char(copy=False)
+
+    @api.constrains("vat")
```
Comment (acsonefho) [inline partner_vat_unique/models/res_partner.py]: Missing `parent_id`.

**Review by acsonefho — CHANGES_REQUESTED** [review]

**Comment by sbidoul** [comment]
I confirm @anizr is covered by the ACSONE ECLA.

File: partner_vat_unique/__manifest__.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Partner VAT Unique",
     "summary": "Module to make the VAT number unique for customers and suppliers.",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
```
Comment (baimont) [inline partner_vat_unique/__manifest__.py:7]: hi @AnizR !

nice pr here. just one comment : I think this kind of change should belong to a commit that says "Migration to 16.0"

**Review by baimont — APPROVED** [review]
code review, thanks @AnizR !

**Review by acsonefho — APPROVED** [review]

**Pull request merged**